adaptive.css

```diff
@@ -0,0 +1,89 @@
+@media (max-width: 375px) {
+  .logo__circle logo__circle_color_dark-blue,
+  .logo__circle logo__circle_color_blue {
+    margin-top: 25px;
+  }
+
+  .main {
+    padding: 34% 0 0;
+    background-size: 34%;
+    background-position: top center;
+  }
+
+  .main__title {
+    font-size: 18px;
+    line-height: 25px;
+    margin-bottom: 20;
+  }
+
+  .main__subtitle {
+    font-size: 15px;
+    line-height: 20px;
+  }
+
+  .main__button {
+    margin: -7px auto 0 -12px;
+  }
+}
+
+@media (max-width: 768px) {
+  .menu {
+    display: none;
+  }
+
+  .header {
+    margin-bottom: 28px;
+  }
+
+  .logo__shadow {
+  display: none;
```

Comment:
Codacy found an issue: [Expected indentation of 4 spaces (indentation)](https://app.codacy.com/gh/WebHeroSchool/LessonsHTML/pullRequest?prid=6240796)